overlord/state/notices_test.go

```diff
@@ -337,13 +331,13 @@ func (s *noticesSuite) TestNoticesFilterType(c *C) {
 	// Multiple types
 	notices = st.Notices(&state.NoticeFilter{Types: []state.NoticeType{
 		state.ChangeUpdateNotice,
-		state.RefreshInhibitNotice,
+		state.RequestsPromptNotice,
```

Comment:
why not have all 3 types used here?

Reply:
The idea was to just check some of the types. But no particular reason not to include `RefreshInhibitNotice` as well. This change is from https://github.com/canonical/snapd/pull/14269 by the way, it will be eaten when that PR merges and this one is rebased.